mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -500,8 +500,13 @@ def execute(self):
             if self.options.skip_flashing:
                 self.logger.prn_inf("copy image onto target... SKIPPED!")
             else:
-                self.logger.prn_inf("copy image onto target...")
-                result = self.mbed.copy_image()
+                result = None
+                for cnt in range(1, 4):
```

Comment:
I would recommend breaking this out as a CLI parameter, not just hard coding 3 tries.

I realize that was my bad code so I guess this is cleaning time 😄